creusot-contracts/src/std/io.rs

```diff
@@ -0,0 +1,18 @@
+use crate::*;
+use ::std::fmt;
+
+extern_spec! {
+    mod std {
+        mod io {
+            // This is an implementation detail of `std`: we specify it so that we can use
+            // `print!` and `println!`.
+            #[ensures(true)]
+            fn _print(args: fmt::Arguments<'_>) {}
```

Comment:
Do we use `println!` or `print!` somewhere in verified code?

It's a bit bold to assume that these functions can be called without VC... This is assuming, for example, the formatting functions do never fail.

Reply:
i have to pretty strongly disagree here. the implication is that you simply cannot use `print` in verified code since there will be no effective way to specify it, no more than we can for OOM errors. We _absolutely_ should support people having print statements in their verified code.

Reply:
I don't know... I wrote thousands of lines of verified code in my life, and I don't think there was any print statement in it!

If one wants to write code that uses IO, then either she needs to specify what it means to perform IO (and what is the global spec of a program that performs IO), or perform IO in unverified code which may call verified functions.

Tracing / debugging print statements is another concern, though. We may provide some infrastructure for this, if we want.